docs/rfcs/XXXX_storage_backends.md

```diff
@@ -0,0 +1,248 @@
+# Storage Backends
+
+**Status:** Approved, waiting for implementation
+
+
+## Background and Existing Solution
+
+![enter image description here](https://i.ibb.co/C7bzPNZ/storage-layout.png)
+> A simple representation of how Data Tracker and its resources are currently allocated.
+
+The Data Tracker relies on a data storage system to both save files that it receives and to account for other files that have been deposited externally. The current method that is used is adequate for a single tenant operating in a single AWS account, but needs to be re-thought for multiple tenants that  have more sophisticated storage needs.
+
+### Study Buckets
+
+Studies are central entities in the Data Tracker. They partition data that is collected and submitted by different research projects and investigators. For this reason, it was determined that each study would have its own bucket to segment it from other collections of data.
+
+#### Creation
+The Data Tracker automatically creates and configures buckets upon creation of a new study. These buckets are created in the same account as the IAM role that the API uses (Kids First Open Data). This account happens to be different from the role where the API actually resides (Kids First STRIDES). This is necessary as compute used to be contained within Kids First Open Data but was migrated to the STRIDES account. However, the data did not move and the codebase required that the IAM used was owned by the same account as the data.
```

Comment:
I know that this paragraph is trying to describe something complicated but it's pretty confusing

Reply:
should `This account happens to be different from the role where the API actually resides (Kids First STRIDES).` say `This IAM role happens to be different from the role where the API actually resides (Kids First STRIDES).`

?